### DISCUSSION TITLE
Google Maps Share Destination Feature?

### DISCUSSION THREAD

**Discussion by brasian11** [BimmerGestalt/AAIdrive#617] [General]
In the readme it states "Google Maps can share destinations to the connected car's navigation". I couldn't figure out how to do this. 

Could someone share the steps in order to enable or use this feature? Thanks!

**Comment by hufman** [comment]
When you have some location's information open in Google Maps, you should see a Share button that brings up an app choser. While the car is connected, you'll see an additional app with AAIdrive's icon and the name BMW or MINI Navigation.
If your car doesn't have a map, you might need to open the Navigation tab and enable Prefer Custom Navigation.
